validator/build.gradle

```diff
@@ -21,13 +21,23 @@ dependencies {
     implementation "commons-io:commons-io:2.4" // not sure why this isn't getting picked up from hri-flink-pipeline-core
     implementation "org.apache.flink:flink-connector-kafka_${scalaBinaryVersion}:${flinkVersion}" // not sure why this isn't getting picked up from hri-flink-pipeline-core
     implementation "com.ibm.fhir:fhir-model:4.7.1"
+    compileOnly("org.scala-lang:scala-compiler") {
+        version {
+            require("${scalaVersion}")
+        }
+    }
 
     //Test dependencies here:
     testImplementation "org.scalactic:scalactic_${scalaBinaryVersion}:${scalaTestVersion}"
     testImplementation "org.scalatest:scalatest_${scalaBinaryVersion}:${scalaTestVersion}"
+    testImplementation("org.scala-lang:scala-compiler") {
```

Comment:
I'm not sure if you need this one since the flink-scala stuff is not pulled in as a testImplementation dependency.